rero_ebooks/config.py

```diff
@@ -47,8 +45,10 @@ def _(x):
 
 # Rate limiting
 # =============
-#: Storage for ratelimiter.
 RATELIMIT_STORAGE_URL = 'redis://localhost:6379/3'
+RATELIMIT_DEFAULT = '5000/second'
+RATELIMIT_ENABLED = False
```

Comment:
does it make sense to have a default rate limit if it is disabled?

Reply:
Yes if we enable it in deployment with an environment variable.